python-avd/pyavd/_eos_designs/schema/schema_fragments/digital_twin.schema.yml

```
@@ -41,6 +41,15 @@ keys:
             description: |-
               Cleartext password for ACT Digital Twin fabric devices.
             default: admin
+          act_backdoor_management:
```

"backdoor" could have strong negative connotations in cybersecurity :D Let's pick something more positive like:
```suggestion
          act_preserve_management_access:
```

---

Maybe `act_ensure_management_access` or `act_ensure_eapi_access`. I think `act_ensure_eapi_access` is the best IMO.

---

Sure, renamed to `act_ensure_eapi_access`